cli/command/context/list.go

```diff
@@ -60,13 +60,13 @@ func runList(dockerCli command.Cli, opts *listOptions) error {
 			Name:               rawMeta.Name,
 			Current:            rawMeta.Name == curContext,
 			Description:        meta.Description,
-			StackOrchestrator:  string(meta.StackOrchestrator),
+			StackOrchestrator:  hideUnset(meta.StackOrchestrator),
```

Comment:
As @thaJeztah said, I think we should remove this "unset" value, instead of trying to hide it.

Reply:
I think we should do that in a followup